modules/tinymce/src/core/test/ts/browser/fmt/RemoveTrailingWhitespaceFormatTest.ts

```diff
@@ -29,15 +29,15 @@ describe('browser.tinymce.core.fmt.RemoveTrailingWhitespaceFormatTest', () => {
   it('unlink with leading whitespace', () => {
     const editor = hook.editor();
     editor.setContent('<p><a href="#">a b</a></p>');
-    TinySelections.setSelection(editor, [ 0, 0, 0 ], 1, [ 0, 0, 0 ], 3);
+    TinySelections.setSelection(editor, [ 0, 0, 0 ], 2, [ 0, 0, 0 ], 4);
```

Comment:
As otherplace, normalization of the content slightly changes it, triggered by nodeChange-event.

Reply:
It mostly just means we move things to be standardized, doesn't it? That's a good thing. This is what'd happen whenever any other command ran anyway. Or typing.